ansible/host_vars/bootstrap-01.aws-eu-central-1a.nimbus.mainnet.yml

```
@@ -3,7 +3,7 @@ swap_file_path: '/main.swap'
 swap_file_size_mb: 4096
 
 # WARNING: Since these are Eth 2 bootnodes we need to keep the keys and IPs unchanged.
-beacon_node_netkey: '{{lookup("bitwarden", "nimbus/netkey", field=hostname)}}'
+beacon_node_netkey: '{{lookup("vault", "netkey", field=hostname, stage="beacon-node")}}'
```

`netkey` is not a service, nor is `beacon-node` a stage:
```suggestion
beacon_node_netkey: '{{lookup("vault", "beacon-node/netkey", field=hostname)}}'
```